demo-angular/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "nativescript": {
-        "id": "org.nativescript.demo",
+        "id": "org.nativescript.demotwilio",
```

Comment:
@jannotabamo it should be `demotwilioangular` as the `demo/` app already has the `demotwilio` ID.